README.MD

```diff
@@ -69,8 +71,105 @@ benchmark = rxr.open_rasterio('benchmark_map_two_class_categorical.tif', mask_an
 
 ![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/metric_table.png)
 
-For more details on how to use this software, check out this [notebook
-tutorial](https://github.com/NOAA-OWP/gval/blob/main/notebooks/Tutorial.ipynb).
+### Continuous Example
+
+The same can be done for continuous rasters as shown below (in this case
+only a subset of the default statistics will be run):
+
+``` python
+import gval
+import rioxarray as rxr
+
+candidate = rxr.open_rasterio('livneh_2011_precip.tif', mask_and_scale=True) # VIC
+benchmark = rxr.open_rasterio('prism_2011_precip.tif', mask_and_scale=True) # PRISM
+
+agreement, metric_table = candidate.gval.continuous_compare(
+    benchmark,
+    metrics=[
+        "coefficient_of_determination",
+        "mean_percentage_error",
+        "mean_absolute_percentage_error",
+        "mean_normalized_mean_absolute_error"
+    ]
+)
+```
+
+#### Continuous Outputs
+
+`agreement_map`
+
+![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/continuous_agreement_map.png)
+
+`metric_table`
+
+![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/continuous_metric_table.png)
+
+### Catalog Example
+
+Entire catalogs can be compared in GVAL, which are represented by
+dataframes of maps. The following is a candidate and benchmark catalog
+for continuous datasets:
+
+`candidate_catalog`
+
+![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/candidate_catalog.png)
+
+`benchmark_catalog`
+
+![alt text](https://github.com/NOAA-OWP/gval/raw/main/docs/images/benchmark_catalog.png)
+
+With the following code a comparison of each pair of maps can be run
+with the following code (note the first pair of candidate and benchmark
+maps are single band rasters while the second pair are multiband
+rasters):
+
+``` python
+import pandas as pd
+
+from gval.catalogs.catalogs import catalog_compare
+
+candidate_continuous_catalog = pd.read_csv('candidate_catalog_0.csv')
+benchmark_continuous_catalog = pd.read_csv('benchmark_catalog_0.csv')
+
+arguments = {
+    "candidate_catalog": candidate_continuous_catalog,
+    "benchmark_catalog": benchmark_continuous_catalog,
+    "on": "compare_id",
+    "map_ids": "map_id",
+    "how": "inner",
+    "compare_type": "continuous",
+    "compare_kwargs": {
+        "metrics": (
+            "coefficient_of_determination",
+            "mean_absolute_error",
+            "mean_absolute_percentage_error",
+        ),
+        "encode_nodata": True,
+        "nodata": -9999,
+    },
+    "open_kwargs": {
+        "mask_and_scale": True,
+        "masked": True
+    }
+}
+
+agreement_continuous_catalog = catalog_compare(**arguments)
+```
+
+#### Catalog Outputs
+
+`agreement_map`
```

Comment:
If I remember correctly, agreement maps are written out to file somewhere or thrown in a column in the agreement catalog? Might be worth showing where that is in the example or just refer to notebook for brevity?

Reply:
I went ahead and added catalog level attributes to both candidate and benchmark catalogs.